components/03-modules/cart-list-item/CartListItem.scss

```diff
@@ -0,0 +1,152 @@
+$cart-list-item__padding                 : $spacer--medium 0 !default;
+$cart-list-item__border                  : 1px solid $gray-light !default;
+$cart-list-item__image-width             : 160px !default;
+$cart-list-item__image-padding           : 0 $spacer--medium 0 0 !default;
+$cart-list-item__name-weight\@medium     : $font-weight-bold !default;
+$cart-list-item__name-width              : 65% !default;
+$cart-list-item__name-padding\@medium    : 0 $spacer--medium 0 0 !default;
+$cart-list-item__attributes-margin       : 0 0 $spacer--medium 0 !default;
+$cart-list-item__qty-width               : 100% !default;
+$cart-list-item__qty-width\@medium       : auto !default;
+$cart-list-item__qty-input-width\@medium : 70px !default;
+$cart-list-item__actions-item-margin     : 0 0 0 $spacer !default;
+$cart-list-item__actions-item-padding    : 0 $spacer--medium 0 0 !default;
+$cart-list-item__button-border           : none !default;
+$cart-list-item__button-background       : none !default;
+$cart-list-item__action-button-background: $gray-darker !default;
+$cart-list-item__action-icon-fill-color  : $gray-darker !default;
+
+.cart-list-item {
+  display: flex;
+  width: 100%;
+  flex-flow: row nowrap;
+  padding: $cart-list-item__padding;
+  border-bottom: $cart-list-item__border;
+
+  &__image {
+    flex: 0 0 35%;
+    max-width: $cart-list-item__image-width;
+    padding: $cart-list-item__image-padding;
+  }
+
+  &__info {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    flex: 1;
+  }
+
+  &__data {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  &__name {
+    margin-bottom: 8px;
+    line-height: $font-line-height;
+    text-decoration: none;
+
+    @include mq($screen-m) {
+      padding: $cart-list-item__name-padding\@medium;
+      font-weight: $cart-list-item__name-weight\@medium;
+    }
+
+    &:hover,
+    &:focus {
+      text-decoration: none;
+    }
+  }
+
+  &__attributes {
+    margin: $cart-list-item__attributes-margin;
+  }
+
+  &__price-value {
```

Comment:
shouldn't be `__price` or `__price-wrapper`?

Reply:
`__price wrapper` sound good, but what do you think abut `__wrapper`? Now this namespace is free to use - regarding earlier changes.